app/controllers/v0/sign_in_controller.rb

```diff
@@ -342,6 +337,10 @@ def idme_auth_service(type)
         @idme_auth_service
       end
     end
+   
+    def mocked_authentication_auth_service
+      @mocked_authentication_auth_service ||= MockedAuthentication::SignIn::Service.new
```

Comment:
This should match the class. 
```suggestion
      @mocked_authentication_auth_service ||= MockedAuthentication::MockedCredential::Service.new
```
(or `Vamock` if you change it)